src/Tests/SFA.DAS.ApprenticeFeedback.Api.UnitTests/Controllers/ApprenticeFeedbackResult/WhenGettingApprenticeFeedbackRatingSummary.cs

```diff
@@ -24,7 +25,7 @@ public async Task And_MediatorCommandIsSuccessful_Then_ReturnOk(
             [Greedy] ApprenticeFeedbackResultController controller)
         {
             mediator.Setup(m => m.Send(It.IsAny<GetApprenticeFeedbackRatingSummaryQuery>(), It.IsAny<CancellationToken>())).ReturnsAsync(summaryResult);
-            var result = await controller.GetApprenticeFeedbackRatingSummary();
+            var result = await controller.GetApprenticeFeedbackRatingSummary(It.IsAny<string>());
```

Comment:
You should not call It.IsAny<string>() here that is for Setup code - pass a real value instead

Reply:
Done